lib/airbrake-ruby/notifier.rb

```diff
@@ -126,7 +126,9 @@ def send_notice(exception, params, sender = default_sender)
       sender.send(notice)
     end
 
-    def default_sender
+    def select_sender
+      @async_sender.spawn_workers unless @async_sender.has_workers?
+
       if @async_sender.has_workers?
```

Comment:
So now you
- spawn workers
- immediately check if there are workers

Sounds wrong and dangerous if workers die sending notices:
1. spawn workers
2. send async
3. workers die and notices are not sent
4. go to step 1